### PR TITLE
dcc: Whitespace fixes

### PR DESCRIPTION
**Pull request by essandess**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 12.1 21C52 arm64
Xcode 13.2 13C90

macOS 11.6.1 20G224 x86_64
Xcode 13.2 13C90

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: mail/dcc/Portfile [hunk 1]
```diff
@@ -5,30 +5,35 @@ PortSystem          1.0
 name                dcc
 version             2.3.168
 revision            0
-checksums           rmd160  2b44aafba4acae1ce34c2485a6a0753e48ecbba2 \
-                    sha256  3fc932325b36a46a93258bdaa483d00ee3a826bea1d00de04f6e84cfbea63bc2 \
-                    size    1608203
-
 categories          mail
-platforms           darwin
+supported_archs     arm64 x86_64
```
Comment (reneeotten) [inline mail/dcc/Portfile:9]: why is this needed? It seems to build fine on i386 as well, at least according to the [status page](https://build.macports.org/builders/ports-10.6_i386-builder/builds/47806)

Comment (essandess) [reply mail/dcc/Portfile:9]: > why is this needed? It seems to build fine on i386 as well, at least according to the [status page](https://build.macports.org/builders/ports-10.6_i386-builder/builds/47806)

It also builds a native M1 binary. I thought ports were supposed to specify which architectures are supported so that the installer knows which binary to grab.

Comment (essandess) [reply mail/dcc/Portfile:9]: > > why is this needed? It seems to build fine on i386 as well, at least according to the [status page](https://build.macports.org/builders/ports-10.6_i386-builder/builds/47806)
> 
> It also builds a native M1 binary. I thought ports were supposed to specify which architectures are supported so that the installer knows which binary to grab.

For example, not having `supported_archs` specified causes issues in e.g. `sf-pwgen`. See
* #13356
* https://trac.macports.org/ticket/63944
* https://trac.macports.org/ticket/64063

Please let me know if this isn’t the correct approach to address correct installs on {{{arm64}}} because there are several ports that are running natively on M1 without this in the Portfile, and I had it in mind to start migrating these to be explicitly native M1 and Intel ports.

Comment (reneeotten) [reply mail/dcc/Portfile:9]: my understanding is that you only need to supply this field if not all the architectures are supported. For example in this case, you can see from the status page I linked above that it builds fine on 10.6_i386. With the change you made here it will now error out on that platform saying that `i386` is not supported. So this isn't the correct approach as far as I can tell. In fact, after I merged the `sf-pwgen` PR is started doubting this was actually correct, but there the port didn't build on i386 according to the status page. 

What the correct solutions is to the tickets you mention I cannot tell you unfortunately.... hopefully someone with more knowledge of base can assist you with this. It seems to me there wasn't a consensus reached on what the solution is in these Trac tickets, so that should probably happen first.

Comment (essandess) [reply mail/dcc/Portfile:9]: Thanks for the explanation. But I’m honestly still confused by it.

When I encountered similar issues installing {{{py-tensorflow-metal}}} on M1, Ryan had me issue a {{{platform i386}}} (and {{{platform arm}}}) directive. See: https://trac.macports.org/ticket/63776#comment:1.

I take it that {{{i386}}} is the MacxPorts “platform,” and {{{x86_64}}} is the MacPorts “architecture”?

If so, wouldn’t this PR build just fine on a platform {{{i386}}} machine?

Comment (essandess) [reply mail/dcc/Portfile]: I removed the `supported_archs` and just kept the minor formatting changes.

**Comment by reneeotten** [comment]
@essandess if you only want to make these whitespace changes then it should be all good, or is there a reason the PR is still marked as "draft"?
